spring-data-microservice-transaction-sample/README.md

```diff
@@ -371,46 +372,42 @@ To stop Cassandra, MySQL and the Microservices, run the following command:
 $ docker-compose down
 ```
 
-## How the microservice transaction is achieved
-
-So far, you have run the sample application, but you haven't seen how it is implemented.
-Let's look at the code to see how the transaction that spans the services is implemented.
+## Reference - How the microservice transaction is achieved
 
-Transaction #2 (Placing an order) achieves the microservice transaction, so we focus on this transaction here.
+The transactions for placing an order, getting an order and getting orders achieve the microservice transaction, so this section focuses on how the transactions that span the Customer Service and the Order Service are implemented by taking placing an order as an example.
```

Comment:
> [!NOTE]
> 
> The following suggestion is the same as the one I mentioned in [microservice-transaction-sample.md](#).

I'd like to clarify this sentence as follows since readers might think `getting an order and getting orders` is a typo. Also, I split this sentence into two since it's become a bit long and includes more details.

```suggestion
The transactions for placing an order, getting a single order, and getting the history of orders achieve the microservice transaction. This section focuses on how the transactions that span the Customer Service and the Order Service are implemented by placing an order as an example.
```